repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -320,9 +330,10 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
       def getTenantScopedRoles(roles: Seq[Role]): Seq[Role] = {
         Option(config.getTenantHandling.getValidateTenant) match {
           case Some(validateTenant) if !validateTenant.isEnableLegacyRolesMode =>
+            val uriTenant = tenantFromUriOption.get
```

Comment:
This `.get` potentially changes the semantics of this method. Rather than always returning `true` or `false`, it may now throw a `NoSuchElementException` which will result in the filter returning a `500` response. Now, we should be protected because if `tenantFromUriOption` would be `None`, then an `UnparseableTenantException` should have already been thrown. The way it was before, that would not be a concern -- you would just end up with an empty `Seq`. Using `.get` also adds another line of code.

So, functionally, the behavior of the filter should not change. The behavior of this method changes slightly, but that likely won't be observed. Do we want to keep this change?